index.js

```diff
@@ -625,15 +625,16 @@ Tree.prototype.copy = function (node, to, transformer, expandAncestors = true) {
         if (i === 0) {
           // Top node in the subtree (node that is being copied)
           if (_to) {
-            ;(_to._allChildren || (_to._allChildren = [])).push(d)
+            let children = (_to._allChildren || (_to._allChildren = []))
```

Comment:
Are the outer parentheses on this line unnecessary?

Reply:
Yea